packages/@aws-cdk/aws-pipes-alpha/test/integ.pipe.ts

```diff
@@ -16,8 +18,8 @@ const enrichmentLambda = new cdk.aws_lambda.Function(stack, 'EnrichmentLambda',
   runtime: cdk.aws_lambda.Runtime.NODEJS_LATEST,
 });
 
-const loggroup = new cdk.aws_logs.LogGroup(stack, 'LogGroup');
-
+// When this module is promoted from alpha, TestSource should
+// be replaced with SqsSource from @aws-cdk/aws-pipes-sources-alpha
```

Comment:
You could make this easier to find later by throwing an error when the package name changes (either checking `__dirname` or `import {name} from '../../package.json'` should be fine)

Reply:
I'm not following you here. Do you have an example?

Reply:
This is probably the cleanest way to do it:

```ts
import { name } from '../package.json';

if (!name.endsWith('-alpha')) {
  throw new Error('aws-pipes has exited alpha, TestSource should now be replaced with SqsSource from @aws-cdk/aws-pipes-sources-alpha');
}
```